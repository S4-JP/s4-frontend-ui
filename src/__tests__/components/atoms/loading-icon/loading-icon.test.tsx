import React from 'react'
import { render } from '@testing-library/react'
import { LoadingIcon } from '../../../../index'

describe('LoadingIconのレンダリング確認', () => {
  test('propsに設定した値でレンダリングされる。', () => {
    const { getByTestId } = render(<LoadingIcon />)

    expect(getByTestId('atoms-li-wrap')).toMatchSnapshot()
  })
})
