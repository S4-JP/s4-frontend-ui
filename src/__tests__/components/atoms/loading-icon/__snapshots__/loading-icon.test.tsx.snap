// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIconのレンダリング確認 propsに設定した値でレンダリングされる。 1`] = `
<div
  class="sc-fXynhf xNuzX"
  data-testid="atoms-li-wrap"
>
  <div
    style="width: 300px; height: 300px;"
  >
    <svg
      height="800"
      preserveAspectRatio="xMidYMid meet"
      style="width: 100%; height: 100%; transform: translate3d(0,0,0);"
      viewBox="0 0 800 800"
      width="800"
      xmlns="http://www.w3.org/2000/svg"
    >
      <defs>
        <clippath
          id="__lottie_element_2"
        >
          <rect
            height="800"
            width="800"
            x="0"
            y="0"
          />
        </clippath>
      </defs>
      <g
        clip-path="url(#__lottie_element_2)"
      >
        <g
          style="display: none;"
        >
          <rect
            fill="#ffffff"
            height="800"
            width="800"
          />
        </g>
        <g
          opacity="1"
          style="display: block;"
          transform="matrix(1,0,0,1,528.8930053710938,400)"
        >
          <g
            opacity="1"
            transform="matrix(1,0,0,1,-216.14500427246094,-5.271999835968018)"
          >
            <path
              d=" M0,-19.11050033569336 C10.54708480834961,-19.11050033569336 19.11050033569336,-10.54708480834961 19.11050033569336,0 C19.11050033569336,10.54708480834961 10.54708480834961,19.11050033569336 0,19.11050033569336 C-10.54708480834961,19.11050033569336 -19.11050033569336,10.54708480834961 -19.11050033569336,0 C-19.11050033569336,-10.54708480834961 -10.54708480834961,-19.11050033569336 0,-19.11050033569336z"
              fill="rgb(0,75,150)"
              fill-opacity="1"
            />
            <path
              d=" M0,-19.11050033569336 C10.54708480834961,-19.11050033569336 19.11050033569336,-10.54708480834961 19.11050033569336,0 C19.11050033569336,10.54708480834961 10.54708480834961,19.11050033569336 0,19.11050033569336 C-10.54708480834961,19.11050033569336 -19.11050033569336,10.54708480834961 -19.11050033569336,0 C-19.11050033569336,-10.54708480834961 -10.54708480834961,-19.11050033569336 0,-19.11050033569336z"
              fill-opacity="0"
              stroke="rgb(255,255,255)"
              stroke-linecap="butt"
              stroke-linejoin="miter"
              stroke-miterlimit="4"
              stroke-opacity="1"
              stroke-width="0"
            />
          </g>
        </g>
        <g
          opacity="1"
          style="display: block;"
          transform="matrix(1,0,0,1,578.3930053710938,400)"
        >
          <g
            opacity="1"
            transform="matrix(1,0,0,1,-216.14500427246094,-5.271999835968018)"
          >
            <path
              d=" M0,-19.11050033569336 C10.54708480834961,-19.11050033569336 19.11050033569336,-10.54708480834961 19.11050033569336,0 C19.11050033569336,10.54708480834961 10.54708480834961,19.11050033569336 0,19.11050033569336 C-10.54708480834961,19.11050033569336 -19.11050033569336,10.54708480834961 -19.11050033569336,0 C-19.11050033569336,-10.54708480834961 -10.54708480834961,-19.11050033569336 0,-19.11050033569336z"
              fill="rgb(130,213,246)"
              fill-opacity="1"
            />
            <path
              d=" M0,-19.11050033569336 C10.54708480834961,-19.11050033569336 19.11050033569336,-10.54708480834961 19.11050033569336,0 C19.11050033569336,10.54708480834961 10.54708480834961,19.11050033569336 0,19.11050033569336 C-10.54708480834961,19.11050033569336 -19.11050033569336,10.54708480834961 -19.11050033569336,0 C-19.11050033569336,-10.54708480834961 -10.54708480834961,-19.11050033569336 0,-19.11050033569336z"
              fill-opacity="0"
              stroke="rgb(255,255,255)"
              stroke-linecap="butt"
              stroke-linejoin="miter"
              stroke-miterlimit="4"
              stroke-opacity="1"
              stroke-width="0"
            />
          </g>
        </g>
        <g
          opacity="1"
          style="display: block;"
          transform="matrix(1,0,0,1,627.8930053710938,400)"
        >
          <g
            opacity="1"
            transform="matrix(1,0,0,1,-216.14500427246094,-5.271999835968018)"
          >
            <path
              d=" M0,-19.11050033569336 C10.54708480834961,-19.11050033569336 19.11050033569336,-10.54708480834961 19.11050033569336,0 C19.11050033569336,10.54708480834961 10.54708480834961,19.11050033569336 0,19.11050033569336 C-10.54708480834961,19.11050033569336 -19.11050033569336,10.54708480834961 -19.11050033569336,0 C-19.11050033569336,-10.54708480834961 -10.54708480834961,-19.11050033569336 0,-19.11050033569336z"
              fill="rgb(0,157,224)"
              fill-opacity="1"
            />
            <path
              d=" M0,-19.11050033569336 C10.54708480834961,-19.11050033569336 19.11050033569336,-10.54708480834961 19.11050033569336,0 C19.11050033569336,10.54708480834961 10.54708480834961,19.11050033569336 0,19.11050033569336 C-10.54708480834961,19.11050033569336 -19.11050033569336,10.54708480834961 -19.11050033569336,0 C-19.11050033569336,-10.54708480834961 -10.54708480834961,-19.11050033569336 0,-19.11050033569336z"
              fill-opacity="0"
              stroke="rgb(255,255,255)"
              stroke-linecap="butt"
              stroke-linejoin="miter"
              stroke-miterlimit="4"
              stroke-opacity="1"
              stroke-width="0"
            />
          </g>
        </g>
        <g
          opacity="1"
          style="display: block;"
          transform="matrix(1,0,0,1,678.3930053710938,400)"
        >
          <g
            opacity="1"
            transform="matrix(1,0,0,1,-216.14500427246094,-5.271999835968018)"
          >
            <path
              d=" M0,-19.11050033569336 C10.54708480834961,-19.11050033569336 19.11050033569336,-10.54708480834961 19.11050033569336,0 C19.11050033569336,10.54708480834961 10.54708480834961,19.11050033569336 0,19.11050033569336 C-10.54708480834961,19.11050033569336 -19.11050033569336,10.54708480834961 -19.11050033569336,0 C-19.11050033569336,-10.54708480834961 -10.54708480834961,-19.11050033569336 0,-19.11050033569336z"
              fill="rgb(0,75,150)"
              fill-opacity="1"
            />
            <path
              d=" M0,-19.11050033569336 C10.54708480834961,-19.11050033569336 19.11050033569336,-10.54708480834961 19.11050033569336,0 C19.11050033569336,10.54708480834961 10.54708480834961,19.11050033569336 0,19.11050033569336 C-10.54708480834961,19.11050033569336 -19.11050033569336,10.54708480834961 -19.11050033569336,0 C-19.11050033569336,-10.54708480834961 -10.54708480834961,-19.11050033569336 0,-19.11050033569336z"
              fill-opacity="0"
              stroke="rgb(255,255,255)"
              stroke-linecap="butt"
              stroke-linejoin="miter"
              stroke-miterlimit="4"
              stroke-opacity="1"
              stroke-width="0"
            />
          </g>
        </g>
      </g>
    </svg>
  </div>
</div>
`;
